app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Welcome#index</h1>
```

Comment:
I'm upping the ante! I want a Frontend caliber homepage before I accept this pull request.